src/node_external_reference.h

```diff
@@ -56,6 +56,13 @@ using CFunctionWithInt64Fallback = void (*)(v8::Local<v8::Value>,
                                             v8::FastApiCallbackOptions&);
 using CFunctionWithBool = void (*)(v8::Local<v8::Value>, bool);
 
+using CFunctionBufferCopy = uint32_t (*)(v8::Local<v8::Value> receiver,
+                                         const v8::FastApiTypedArray<uint8_t>& source,
+                                         const v8::FastApiTypedArray<uint8_t>& target,
```

Comment:
Can you avoid using this? It's been deprecated in V8 12.8

Reply:
What's the alternative? Using `Local<Value>` is much slower?

Reply:
We have other places where we need to change it as well. So could maybe resolved in one PR that fixes it everytwhere.

Reply:
I don't know what's the alternative.

Reply:
Just know that I will have to revert this change in the V8 update PR.

Reply:
It's deprecated. Doesn't that mean that it will still work? What will you do about e.g. `FastCompare`?

Reply:
As I see it this can't be fixed pre 12.8. They write that on should use `Local<Value>` but that doesn't seem to be supported pre 12.8.

Reply:
```
signature should be `v8::Local<v8::Value>` instead of "
    "FastApiTypedArray<>. The API function then has to type-check the "
    "parameter and convert it to a `v8::Local<v8::TypedArray` to access the "
    "data. In essence, the parameter should be handled the same as for a "
    "regular API call."
```

Reply:
> What will you do about e.g. `FastCompare` and `FastIndexOfNumber`?

See https://github.com/v8/node/pull/197 which I already backported in my PR.

They say there's no advantage to using fast API calls in these cases.

Reply:
Intersting. That's not what my benchmarks tell me. Was there any benchmark runs before that landed?

Reply:
At least for this case the perf difference is huge:

```
buffers/buffer-copy.js n=6000000 partial='false' bytes=0                     2.75 %       ±4.57% ±6.16% ±8.16%
buffers/buffer-copy.js n=6000000 partial='false' bytes=128          ***    -60.61 %       ±5.01% ±6.74% ±8.92%
buffers/buffer-copy.js n=6000000 partial='false' bytes=32768                 1.60 %       ±2.01% ±2.67% ±3.48%
buffers/buffer-copy.js n=6000000 partial='false' bytes=8            ***    -60.67 %       ±3.35% ±4.50% ±5.94%
buffers/buffer-copy.js n=6000000 partial='true' bytes=0                     -2.59 %       ±4.95% ±6.60% ±8.62%
buffers/buffer-copy.js n=6000000 partial='true' bytes=128           ***     10.98 %       ±4.36% ±5.80% ±7.54%
buffers/buffer-copy.js n=6000000 partial='true' bytes=32768                 -0.06 %       ±2.30% ±3.07% ±4.02%
buffers/buffer-copy.js n=6000000 partial='true' bytes=8              **      7.30 %       ±4.22% ±5.61% ±7.31%
```

Reply:
> It's deprecated. Doesn't that mean that it will still work?

You're right. It will be eventually removed but I don't need to handle the deprecation in the next V8 update.

Reply:
Then it's a bit of a stale mate. I can't remove it until there is an alternative. It's kind of sad not applying such a significant performance improvement.

However, if you object to landing this now and solving it later when there maybe/hopefully is an alternative before the api is totally removed, then I think we should close this PR.

Reply:
I'm a bit sceptical about https://github.com/v8/node/pull/197. Has that landed already in `nodejs/node` or is there a PR we can benchmark?

Reply:
I don't know if I'm doing it right, but here is a PR that replaces the use of `FastApiTypedArray` with `Local<Object>`: https://github.com/nodejs/node/pull/54103

Reply:
Fixed. Thanks!